estate/models/estate_property_offer.py

```diff
@@ -43,3 +52,16 @@ def action_accept(self):
     def action_refuse(self):
         for record in self:
             record.status = 'refused'
+
+    @api.model
+    def create(self, vals_list):
+        for val in vals_list:
```

Comment:
```suggestion
        for vals in vals_list:
```
it goes like vals_list -> vals -> val
vals_list refers to a list of dictionaries, every dictionary consists of multiple values.